.github/workflows/performance.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
             - name: Compare performance with trunk
               if: github.event_name == 'pull_request'
-              run: ./bin/plugin/cli.js perf $GITHUB_SHA trunk --tests-branch $GITHUB_SHA
+              run: ./bin/plugin/cli.js perf trunk
```

Comment:
How do we ensure that we run the performance tests that are on the PR. For instance when we make changes to the tests themselves?

Reply:
Oh If I'm reading this, there's now an assumption that the perf cli.js is included within a Gutenberg plugin and that we need we compare with the current git hash automatically and use it also as tests branch.

One of the initial goal of the cli is to actually be "independent", while it lives in the Gutenberg plugin, it doesn't have any requirement in the sense that the current branch could be broken, have edits to Gutenberg code... it doesn't matter, the tool would always perform a clone. In fact one goal was to make it independent of Gutenberg entirely and be a tool any plugin can use. It seems the current PR goes into the opposite direction.

Reply:
For the general direction of this PR, see my answer [here](https://github.com/WordPress/gutenberg/pull/49238#discussion_r1168419640). 

> (…) In fact one goal was to make it independent of Gutenberg entirely and be a tool any plugin can use.

I'm struggling to visualize how the perf tests CLI runner, as an independent tool, would be utilized by other plugins. Would it not be running from Gutenberg then, but would still be comparing Gutenberg branches? It would be really helpful if you could give an example of a plugin development flow where this tool is used 🙏

Reply:
Yeah, the initial goal was to be an independent tool, something like so:

```
# installing the tool
npm install -g @wordpress/plugin-cli

# writing a config file
touch { repoUrl: "some url", buildCommand: "npm run build", perfCommand: "npm run perf", wpPluginSlug: "someslug" } >> plugin-cli.json

# running the perf command
plugin-cli  ---config plugin-cli.json some-branch some-other-branch
```

The goal was also for the cli to include more than just perf command, maintaining changeling, releasing to SVN/Github was also a command that was part of the same tool, I see that it was removed now in favor of all the git workflows. I'm kind of sad that it didn't remain as a command to be used by the workflows instead of writing it within the workflow files like we do now.

Anyway, given that the previous direction is also to move away from the goal of being an independent tool, one more step away won't be that harmful here, so I guess feel free to make that change if you think it's a good idea.

Reply:
Sounds good to me 👍